tsconfig.json

```diff
@@ -1,111 +1,19 @@
 {
-  "files": [],
-  "include": [],
-  "references": [
-    {
-      "path": "./packages/badge"
-    },
-    {
-      "path": "./packages/banner"
-    },
-    {
-      "path": "./packages/box"
-    },
-    {
-      "path": "./packages/button"
-    },
-    {
-      "path": "./packages/callout"
-    },
-    {
-      "path": "./packages/card"
-    },
-    {
-      "path": "./packages/checkbox"
-    },
-    {
-      "path": "./packages/code"
-    },
-    {
-      "path": "./packages/confirmation-modal"
-    },
-    {
-      "path": "./packages/emotion"
-    },
-    {
-      "path": "./packages/hooks"
-    },
-    {
-      "path": "./packages/icon"
-    },
-    {
-      "path": "./packages/icon-button"
-    },
-    {
-      "path": "./packages/lib"
-    },
-    {
-      "path": "./packages/logo"
-    },
-    {
-      "path": "./packages/menu"
-    },
-    {
-      "path": "./packages/modal"
-    },
-    {
-      "path": "./packages/mongo-nav"
-    },
-    {
-      "path": "./packages/palette"
-    },
-    {
-      "path": "./packages/pipeline"
-    },
-    {
-      "path": "./packages/popover"
-    },
-    {
-      "path": "./packages/portal"
-    },
-    {
-      "path": "./packages/radio-box-group"
-    },
-    {
-      "path": "./packages/radio-group"
-    },
-    {
-      "path": "./packages/side-nav"
-    },
-    {
-      "path": "./packages/stepper"
-    },
-    {
-      "path": "./packages/syntax"
-    },
-    {
-      "path": "./packages/table"
-    },
-    {
-      "path": "./packages/testing-lib"
-    },
-    {
-      "path": "./packages/text-input"
-    },
-    {
-      "path": "./packages/theme"
-    },
-    {
-      "path": "./packages/toggle"
-    },
-    {
-      "path": "./packages/tooltip"
-    },
-    {
-      "path": "./packages/leafygreen-provider"
-    },
-    {
-      "path": "./packages/typography"
-    }
-  ]
+  "include": ["packages/**/src/**/*", "typings/**/*"],
+  "compilerOptions": {
```

Comment:
Anyway we can extend from package.tsconfig.json as the settings should be mostly the same? Or vice versa?